### DISCUSSION TITLE
Adetailer installation

### DISCUSSION THREAD

**Discussion by carlmoss22** [MoonRide303/Fooocus-MRE#176] [Ideas]
i played around with fooocus-mre and i have to say i really like it! well done!

because faces are usually a problem( for my style) it would be a massive helper to have adetailer working.

i don't know whether how much work it is to implement it, but it would be a massive helper  and time safer.

**Comment by 2PeteShakur** [comment]
yes it would be nice too, i would suggest also - but dunno when there will be updates to this branch, it's been a while.
